rules/camunda-cloud/escalation-reference.js

```diff
@@ -62,6 +62,6 @@ module.exports = skipInNonExecutableProcess(function() {
   };
 });
 
-function isNoEscalationRefAllowed(node, version) {
-  return is(node, 'bpmn:BoundaryEvent');
+function isNoEscalationRefAllowed(node) {
+  return isAny(node, [ 'bpmn:CatchEvent', 'bpmn:BoundaryEvent' ]);
```

Comment:
[weak suggestion] Perhaps `!is(node, 'bpmn:ThrowEvent)` shows the intent better. We require escalation reference if the event is to trigger escalation, because we need to know what escalation is about to be triggered.